src/Misc/environmentTextureTools.ts

```diff
@@ -616,9 +694,7 @@ export class EnvironmentTextureTools {
      * @param info defines the environment texture info retrieved through the GetEnvInfo method
      */
     public static UploadEnvSpherical(texture: InternalTexture, info: EnvironmentTextureInfo): void {
-        if (info.version !== 1) {
-            Logger.Warn('Unsupported babylon environment map version "' + info.version + '"');
-        }
+        info = EnvironmentTextureTools.normalizeEnvInfo(info);

```

Comment:
Note that this *could* be seen as some kind of breaking change, as an unsupported version will now throw, while previously it only issued a warning. But I believe this was wrong, as the schema could have been totally different. At least it was inconsistent with the other places where an error was already thrown...